### PR TITLE
Update Typings to Adhere to Strict TS

### PR DESCRIPTION
**Pull request by cfanoulis**
What it says on the tin. Also, updated the caniuse library

- refactor: properly type theme object
- chore: update caniuse

### CODE REVIEW

File: packages/theme/src/index.ts [hunk 1]
```diff
@@ -26,7 +26,16 @@ const colors = {
   instagram: '#e1306c'
 }
 
-const theme = {
+const CSS_QUERIES = {
```
Comment (sampoder) [inline packages/theme/src/index.ts:29]:
```suggestion
const cssQueries = {
```

**Review by sampoder — APPROVED** [review]
I'm happy with this PR, @maxwofford thoughts?

**Comment by maxwofford** [comment]
Thanks for the PR @cfanoulis! Is this a version bump for @hackclub/theme?

**Comment by cfanoulis** [comment]
@maxwofford yup, should be a patch bump

**Comment by maxwofford** [comment]
Looks good to merge for me if this doesn't break any existing imports!

**Pull request merged**